src/main/java/io/passport/server/controller/SurveyController.java

```diff
@@ -49,16 +49,21 @@ public ResponseEntity<?> getSurveyById(@PathVariable("surveyId") Long surveyId)
     }
 
     /**
-     * Read surveys by surveyId
-     * @param studyId ID of the study.
+     * Read surveys, if studyId is provided, filter by studyId; otherwise, return all surveys.
+     * @param studyId Optional ID of the study.
      * @return
      */
     @GetMapping()
-    public ResponseEntity<List<Survey>> getSurveysByStudyId(@RequestParam("studyId") Long studyId) {
+    public ResponseEntity<List<Survey>> getSurveys(@RequestParam(value = "studyId", required = false) Long studyId) {
```

Comment:
It's not a big deal, but I think it would be good if we have two separate APIs: getSurveys and getSurvey